content/en/docs/reference/config/istio.operator.v1alpha12.pb/index.html

```diff
@@ -128,7 +128,7 @@
   galley:
     k8s:
       overlays:
-      - apiVersion: extensions/v1beta1
+      - apiVersion: apps/v1
```

Comment:
@ostromart Could you make this corresponding change in the operator sources please?